### DISCUSSION TITLE
服务器推荐IP

### DISCUSSION THREAD

**Discussion by SLZWAD** [Meloong-Git/PCL#3047] [提问] [answered]
做一个类似公告的简易服务器推荐列表没得玩可以玩服务器可以分为正版验证和没有正版验证 有正版:)

**Comment by MoYuan-CN** [comment]
PCL 是启动器……
这个貌似有点超出启动器的范围.......
或许做成预设自定义主页？

**Reply by SLZWAD** [reply]
也可以的启动页面右边有点空

**Reply by SLZWAD** [reply]
做成类似新闻的或许龙猫还可以赚点广告费😂

**Reply by SqrtTan** [reply]
想法很好，龙猫不要

**Comment by Deep-Dark-Forest** [comment]
findmcserver.com试试

**Comment by WForst-Breeze** [comment, marked as answer]
由 #1047 #1993 可以看出，不会制作类似功能，超出了 PCL 的服务范畴了，这应该是第三方自定义主页来实现的内容🤔

**Reply by WorldHim** [reply]
#1751
